File: SigmaBooking.Backend/SigmaBooking.Core/Models/Booking.cs
using System;

namespace SigmaBooking.Core.Models;

public class Booking
{
    public string Id { get; set; }
    public string TableId { get; set; }
    public Table Table { get; set; }
    public string Name { get; set; }
    public string Phone { get; set; }
    public string Email { get; set; }
    public int PeopleCount { get; set; }
    public string Date { get; set; }
    public string StartTime { get; set; }
    public string EndTime { get; set; }
    public bool IsEating { get; set; }
    public string Description { get; set; }
}

File: SigmaBooking.Frontend/src/models/Booking.ts
import {Table} from "./Table";

export interface Booking {
    id: string;
    table: Table;
    tableId: string;
    name: string;
    phone: string;
    email: string;
    date: string;
    peopleCount: number;
    startTime: string;
    endTime: string;
    isEating: boolean;
    description: string;
}